contracts/core/ManualVolOracle.sol

```diff
@@ -48,14 +48,15 @@ contract ManualVolOracle is AccessControl {
 
     /**
      * @notice Sets the annualized standard deviation of the base currency of the `pool`
-     * @param pool is the uniswap pool we want to set annualized volatility for
-     * @param annualizedVol is the annualized volatility with 10**8 decimals i.e. 1*10**8 = 100%
+     * @param _pool is the uniswap pool we want to set annualized volatility for
+     * @param _annualizedVol is the annualized volatility with 10**8 decimals i.e. 1*10**8 = 100%
      */
-    function setAnnualizedVol(address pool, uint256 annualizedVol)
+    function setAnnualizedVol(address _pool, uint256 _annualizedVol)
         external
         onlyAdmin
     {
-        require(annualizedVol > 0, "!annualizedVol");
-        annualizedVols[pool] = annualizedVol;
+        require(_annualizedVol > 50 * 10**6, "Cannot be less than 50%");
+        require(_annualizedVol < 200 * 10**6, "Cannot be more than 200%");
```

Comment:
these constraints are pretty tight. it could be less than 50% and more than 200% and the code wont actually break if its 49% vol or 201% vol (9200% different story). I would suggest like 10% - 400%

Reply:
The intention here is to keep the lower bound tight, because that's the worst-case scenario where we will select a low strike price. Agree on 400% as upper bound